app/views/components/_task_list.html.erb

```diff
@@ -127,7 +130,7 @@
                   </div>
                 <% end %>
 
-                <% if task_list_url %>
+                <% if task_list_url and step[:show_help_link] %>
```

Comment:
We use `&&` rather than `and`

Reply:
Argh, I knew we preferred one over the other but couldn't remember which one.

Reply:
If we linted our ERB it'd tell you.

Reply:
Yeah, that'd be helpful.